tests/test_frame.py

```diff
@@ -1080,6 +1080,15 @@ def test_frame_getitem_isin() -> None:
     check(assert_type(df[df.index.isin([1, 3, 5])], pd.DataFrame), pd.DataFrame)
 
 
+def test_to_excel() -> None:
+    if TYPE_CHECKING:  # skip pytest
```

Comment:
Why does this need to be skipped? Is it because one of the excel dependencies is not installed?

Reply:
Otherwise openpyxl would be a dependency only for this case. As far as I know it is not needed elsewhere. Do you think thats worth the dependency?

Reply:
I think we can add it as a `dev-dependencies` in the `pyproject.toml` .  Can you try that?

Reply:
I added openpyxl to the dev-dependencies.